src/System.Runtime.Intrinsics.X86/src/System.Runtime.Intrinsics.X86.csproj

```diff
@@ -3,14 +3,38 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Runtime.Intrinsics.X86</AssemblyName>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)'== 'netcoreapp'">true</IsPartialFacadeAssembly>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Unix-Debug|AnyCPU'" />
```

Comment:
I believe you still want to keep the `netcoreapp-Unix` and `netcoreapp-Windows_NT` configurations, so instead of removing the existing, simply add the the new configurations (netstandard debug and release)

Reply:
Thank you! Will fix all the above feedback.